docs/content/policy-language.md

```diff
@@ -3786,65 +3675,13 @@ Compiler Strict mode is supported by the `check` command, and can be enabled thr
-
-## The `rego.v1` Import
-
-In the future, when [OPA v1.0](../opa-1) is released, breaking changes will be introduced to the Rego language.
-The `rego.v1` import is a way to opt-in to these breaking changes early, and ensure that your policies are compatible with OPA v1.0.
-If a module containing this import is not compatible with OPA v1.0, it will cause a compilation error.
-
-When a module imports `rego.v1`, the following features and constraints are implied:
-
-* all [Future keywords](#future-keywords) are implied and can be used without import.
-  These imports are mutually exclusive, and it will cause a compilation error to import both `rego.v1` and `future.keywords` in the same module.
-* the [if](#futurekeywordsif) keyword is required before a rule body declaration.
-* the [contains](#futurekeywordscontains) keyword is required for partial set (multi-value) rules.
-* most [Strict mode](#strict-mode) constraints and checks are implied and enforced. See the strict mode [constraints and checks table](#strict-mode-constraints-and-checks) for details.
-
-The `rego.v1` import only affects the module where it's declared. It does not affect any other modules, even if they are importing, or is imported by, a module where `rego.v1` is declared.
-
-In OPA v1.0, the `rego.v1` import will have no semantic impact on the policy, as all its implied features and constraints will be enforced by default. It will however still be a valid statement, and won't cause any compilation errors.
-
-Example policy that imports `rego.v1` to be compatible with the future syntax in OPA v1.0:
-
-```live:rego_v1:module:read_only
-package example
-
-import rego.v1
-
-l := [1, 2, 3]
-
-default allow := false
-
-# 'if' is part of default v1.0 syntax, and doesn't need import.
-# 'if' is required before rule body.
-allow if {
-  count(violations) == 0
-}
-
-# 'contains' is part of default v1.0 syntax, and doesn't need import.
-# 'contains' is required to declare multi-value, partial set rule.
-violations contains msg if {
-  # 'every' and 'in' are part of default v1.0 syntax, and doesn't need imports.
-  every x in l {
-    x > 0
-  }
-  msg := "no negative entries"
-}
-```
+Name | Description                                                                                                                                                                                                                                                    
+--- |----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+Duplicate imports | Duplicate [imports](../policy-language/#imports), where one import shadows another, are prohibited.                                                                                                                                                            
```

Comment:
We should drop this table entry, as this is part of the default `v1` constraints.

We could keep it, with a comment, to let users know it's still relevant with `--v0-compatible --strict`. But maybe it's more clear if that's a separate table. And then it should probably also be put in the v0 migration guide, with just a link here.

Reply:
I've reworded this section to align with 1.0